apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -530,6 +558,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     * @param _voteId Vote identifier
     * @return True if the vote is within its overrule window
     */
+   // Note: maybe better renamed as `canRepresentativesVote()`?
     function withinOverruleWindow(uint256 _voteId) external view returns (bool) {
```

Comment:
Thought that perhaps we should stick to `can*`-names rather than `isVoteOpen()` and `withinOverruleWindow()`, to align with `canExecute()`, etc. It might also allow us to deduplicate some logic (e.g. `withinOverruleWindow()` is awkwardly not the same as `_canRepresentativesVote()`).

Reply:
It is actually the other way around, during the overrule window representatives cannot vote

Reply:
Yes exactly—so with the above change on changing the configuration to be "start date"-based, `canRepresentativesVote()` seems more meaningful than `withinOverruleWindow()`.

Reply:
Siezed 8765ccf206f6d4e8ff78e51d5e84d9dd7a77810d and renamed it :)